terragrunt/env/production/env_vars.hcl

```
@@ -0,0 +1,6 @@
+inputs = {
+  account_id       = "806721586252"
+  env              = "production"
+  cost_center_code = "url-shortener-production"
+  domain           = ""
```

I think the domain  we're going with to start is `o.alpha.canada.ca`
```suggestion
  domain           = "o.alpha.canada.ca"
```

---

Done!